docs/dashboard/x-ray.md

```diff
@@ -49,38 +49,68 @@ http.clientip = 1.2.3.4
 
 ## Understanding an XRay trace
 
-Each HTTP request is called an "XRay trace" and provides many useful pieces of debugging information.
+Each HTTP request generates an "X-Ray trace" and provides many useful pieces of debugging information.
+
+### Overview
+
+![](../assets/xray-overview.png)
+
+The overview table provides quick information about the request, including requested URL, total duration, response code, and external API calls to the database and remote servers.
```

Comment:
You mentioned this was response time rather than total duration now iirc?

Reply:
That was in the table view, but @joehoyle I guess we need to update the docs to reflect that change too?

Reply:
```suggestion
The overview table provides quick information about the request, including requested URL, response time, response code, and external API calls to the database and remote servers.
```